library/core/src/main/java/com/google/android/exoplayer2/util/MimeTypes.java

```diff
@@ -211,6 +212,9 @@ public static boolean isApplication(@Nullable String mimeType) {
     codec = Util.toLowerInvariant(codec.trim());
     if (codec.startsWith("avc1") || codec.startsWith("avc3")) {
```

Comment:
Hm, this is a bit inconvenient! Do you know whether "avc3.09" can ever appear for a non-DV H.264 stream? If not I guess we can add a special case that checks for that string. Otherwise I suggest we just say that we won't support profile 9 in this change, and figure out how to handle this later.